decentraland/explorer
# feat: Rebuild Builder in-world editor Loading Screen

### What this PR includes?
- [x] The entire world UI disappears from screen to make room for this modal. All shortcuts are locked except the key to exit the editor mode.
- [x] When the LAND loading is completed, the modal disappears to give place to the Builder UI.
- [x] The tips (not yet defined) of this modal are dynamic (use the Loading screen of the client as a reference for the persistence on the screen of each message). 
- [x] The transition from one tip to another is an automatic carousel (the user cannot do back and forward). The tips exit from the left and enter from the right.
- [x] The load bar shall consist of 2 main parts: loading of the catalog (50%) + loading of the scene to edit (50%)

![image.png](https://images.zenhubusercontent.com/603cdb58edb53386a3d0d8d5/61b671c6-3857-4bd1-a417-9cda500c079a)

![image.png](https://images.zenhubusercontent.com/603cdb58edb53386a3d0d8d5/767e0499-59d3-4ed7-a79e-b08fda0709f7)

### Testing
1. Go to: https://explorer.decentraland.zone/branch/feat/Rebuild-Builder-in-world-editor-Loading-Screen/index.html?ENV=zone&ENABLE_BUILDER_IN_WORLD&position=100,100&realm=fenrir-amber
2. Activate the Builder In World editor pressing the "**K**" key.
3. See the new loading screen.
